### DISCUSSION TITLE
Dúvida sobre instalação do ambiente

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#432] [Tira Dúvidas] [answered]
Eu estou assistindo às aulas e li os artigos do blog:
https://balta.io/blog/wsl
https://balta.io/blog/sql-server-docker

Mas ainda sim fiquei com dúvidas:
1 - O docker deve ser instalado **dentro** do WSL? Se sim, eu deveria usar a versão para Ubuntu? Considerando que o subsistema é um Ubuntu.
2 - A seção "Instalando o .NET" no WSL não funcionou, eu ainda tenho o erro de comando não encontrado quando executo "dotnet --version"

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @mlasa como vai?

1 - O docker deve ser instalado dentro do WSL? Se sim, eu deveria usar a versão para Ubuntu? Considerando que o subsistema é um Ubuntu.
- Sim, igual consta no tutorial que mencionou

2 - A seção "Instalando o .NET" no WSL não funcionou, eu ainda tenho o erro de comando não encontrado quando executo "dotnet --version"
- Pode instalar o .NET no Windows mesmo, não precisa rodar ele no WSL.
